examples/pipelines/filter-cc-25m/README.md

```diff
@@ -0,0 +1,83 @@
+# Creative common license dataset
+
+## Overview
+
+We present a sample pipeline that demonstrates how to effectively utilize a creative
+commons image dataset within a fondant pipeline. This dataset comprises images from diverse sources
+and is available in various data formats. In this illustrative example, our objective is to refine
+the dataset to exclusively include PNG files.
+
+We with the initialization of the image dataset sourced from HuggingFace.
+Afterwards, we narrow down the dataset contents to exclusively encompass PNG files. Finally, we
+proceed with the downloading of these carefully selected images. Accomplishing these tasks
+necessitates the use of both pre-built reusable components (HuggingFace dataset loading and image
+downloading) and the creation of a customized component tailored to filter images based on their
+data type.
+
+To better understand the instructions provided, we highly recommend familiarizing yourself with the
+fundamental concepts of Fondant. To do so, you can take a look at
+our [getting started](https://fondant.readthedocs.io/en/stable/getting_started) page.
+
+### File structure
+
+The directory structure is organized as follows:
+
+```
+.
+├── components
+│ ├── download_images
+│ ├── filter_file_type
+│ └── load_from_hf_hub
+└── pipeline.py
+```
+
+Within this folder, you will find the following items:
+
+- pipeline.py: This file defines the core pipeline and serves as the starting point for the pipeline
+  execution.
+- components: This directory contains three distinct components.
+
+The load_from_hub and download_images components are reusable components, while the filter_file_type
```

Comment:
...thus they only require a fondant_component.yaml referencing the reusable component.

(so that one (me) is not surprised to see so few files in these component directories)